DOCS.md

```diff
@@ -92,117 +115,225 @@ This messaging will be displayed to the user for several seconds, after which th
+The `adFreePod` event signifies that the user has earned a credit with TrueX and all linear video ads remaining in the 
+current ad break should be skipped. If the `adFreePod` event did not fire before a terminal event is emitted, 
+the app should resume playback without skipping any ads, so the user receives a normal video ad payload.
 
 #### Terminal Events
 
 The *terminal events* are:
-* `adCompleted`: the user has exited the true[X] ad
+* `adCompleted`: the user has exited the TrueX ad
 * `userCancelStream`: the user intends to exit the current stream entirely
-* `noAdsAvailable`: there were no true[X] ads available to the user
+* `noAdsAvailable`: there were no TrueX ads available to the user
 * `adError`: the renderer encountered an unrecoverable error
 
-It's important to note that the player should not immediately resume playback once receiving the `adFreePod` event -- rather, it should note that it was fired and continue to wait for a terminal event.
-
+It's important to note that the player should not immediately resume playback once receiving the `adFreePod` event 
+-- rather, it should note that it was fired and continue to wait for a terminal event.
 
 ### Handling Ad Elimination
 
-Skipping video ads is completely the responsibility of the host app code. The SSAI API should provide enough information for the host app to determine where the current pod end-point is, and the app, when appropriate, should fast-forward directly to this point when resuming playback.
+Skipping video ads is completely the responsibility of the host app code. The ad provider SDKs like Google IMA should
+provide enough information for the host app to determine where the current pod end-point is, and the app, 
+when appropriate, should fast-forward directly to this point when resuming playback.
 
-## TruexAdRenderer Web API
+## TruexAdRenderer API
 
-This is an outline of `TruexAdRenderer` input and output events. Input events are assigned on the `action` field from the interface of the `TruexAdRenderer`, while output events are emitted against the `event` field on the same component.
+Here we describe the essential methods of the `TruexAdRenderer` class, as well as ad events that can be listened for.
 
-### Installing true[X] Library
+Note that there are actually 3 possible classes to use:
+```javascript
+TruexAdRendererCTV
+TruexAdRendererDesktop
+TruexAdRendererMobile
+```
+One should use the appropriate one for your platform, so as to allow the correct ad tracking to be used.
 
-The true[X] interactive ad component and its rendering logic are distributed as part of an npm package. It is required for the hosting app to include the renderer library in order for it to be used, this can be achieved by running this `npm` command:
+If one uses `TruexAdRenderer`, that maps to `TruexAdRendererCTV`
```

Comment:
Is `TruexAdRenderer` deprecated? If so, you don't need to mention it here, just cut this line. It's confusing

Reply:
It is not deprecated, and now it probably should be.
That tracking category thing is the reason, and it is a little annoying, e.g. fep_ctv vs fep vs fep_mobile
And it only matters for a second, only until the choice card code takes over

The thing is we are using that class currently in all our reference apps and docs.

I need to check that the webview bridge actually does the right thing.

Reply:
it is deprecated now, which is now mentioned.